tests/llm/fixtures/test_ask_holmes/71_connection_pool_starvation/generate_logs.py

```
@@ -88,27 +105,40 @@ def main():
+            print(
+                generate_log_entry(
+                    100, wait_time=random.randint(25000, 30000), sequence_num=seq
+                )
+            )
+            seq += 1
 
     # More normal logs after the incident
-    for i in range(34000):
+    # Was 34,000, now ~677 to reach 2000 total
+    remaining = 2000 - seq
+    for i in range(remaining):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

(B007)